src/data/monsters/index.js

```diff
@@ -34,6 +35,7 @@ const monsters = {
     ghost: Ghost,
     imp: Imp,
     lich: Lich,
+    'lich king': LicKing,
```

Comment:
How come this was changed to “LicKing”? Did you remove the ‘h’ that was here for a reason?